### PR TITLE
Set up test & build flows

### PR DESCRIPTION
**Pull request by SamMousa**
## What does this PR accomplish?

It sets up CI build & test flows.
Tests will be run on macos & linux for every push & PR.
Builds of android and ios will be run for every push to master (this includes merges)

Based on work done by @epicfaace
 
## Did you add any dependencies?
No

## How did you test the change?

By pushing to a branch until it worked here: 
https://github.com/WorldHealthOrganization/app/actions/runs/63788205
https://github.com/WorldHealthOrganization/app/actions/runs/63788201

### CODE REVIEW

File: .github/workflows/flutter-test.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
+      - run: git clone https://github.com/flutter/flutter.git --depth 1 -b stable _flutter
+      - run: echo "::add-path::$GITHUB_WORKSPACE/_flutter/bin"
+      - run: flutter pub get
+        working-directory: ./client/flutter
+      - run: flutter test
+        working-directory: ./client/flutter
+      - run: flutter build apk
```
Comment (epicfaace) [inline .github/workflows/flutter-test.yml]: We can remove this because it's already in the build flow

**Comment by SamMousa** [comment]
@epicfaace does it look good to you now?

File: .github/workflows/flutter-build.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: Flutter build
```
Comment (epicfaace) [inline .github/workflows/flutter-build.yml]: It might be worth it to also add a web build to the CI.

Comment (SamMousa) [reply .github/workflows/flutter-build.yml]: Seeing as the web is still in beta and it is already build & published by netlify I don't see a need for doing that here as well.

**Pull request merged**